manifests/version.pp

```diff
@@ -1,25 +1,42 @@
 # Places SIMP version related information on the filesystem
-class simp::version {
-
-  file { '/etc/simp':
-    ensure => 'directory',
-    owner  => 'root',
-    group  => 'root',
-    mode   => '0644'
+class simp::version () {
+  # XXX: ToDo: Move /etc/simp creation to a simplib class and use
+  # moduledata to resolve the variables..
+  #
+  # It's needed in more places then here, and they don't need to pull in
+  # anything else from simp
+  if (downcase($facts['kernel']) == 'windows') {
+    $simp_root_dir = "C:/ProgramData/SIMP"
```

Comment:
Why are these settings here and not under the data?  There is a kernel option in the hiera.yaml?